benchmarks/huggingface.py

```diff
@@ -28,45 +27,240 @@ def pip_install(package):
     print("Installing HuggingFace Transformers...")
     pip_install("git+https://github.com/huggingface/transformers.git#egg=transformers")
 finally:
-    from transformers import AlbertConfig
     from transformers import AlbertForPreTraining
     from transformers import AutoConfig
     from transformers import AutoModelForCausalLM
     from transformers import AutoModelForMaskedLM
     from transformers import AutoModelForSeq2SeqLM
-    from transformers import BartConfig
-    from transformers import BartForConditionalGeneration
-    from transformers import BertConfig
-    from transformers import BertForPreTraining
     from transformers import BigBirdConfig
-    from transformers import DebertaConfig
-    from transformers import DebertaForMaskedLM
-    from transformers import GPT2Config
-    from transformers import GPT2LMHeadModel
-    from transformers import RobertaConfig
-    from transformers import RobertaForMaskedLM
-    from transformers import T5Config
-    from transformers import T5ForConditionalGeneration
+    from transformers import BlenderbotForConditionalGeneration
+    from transformers import BlenderbotModel
+    from transformers import BlenderbotSmallForConditionalGeneration
+    from transformers import BlenderbotSmallModel
+    from transformers import CLIPModel
+    from transformers import CLIPVisionModel
+    from transformers import ElectraForPreTraining
+    from transformers import GPT2ForSequenceClassification
+    from transformers import GPTJForSequenceClassification
+    from transformers import GPTNeoForSequenceClassification
+    from transformers import HubertForSequenceClassification
+    from transformers import LxmertForPreTraining
+    from transformers import LxmertForQuestionAnswering
+    from transformers import MarianForCausalLM
+    from transformers import MarianModel
+    from transformers import MarianMTModel
+    from transformers import PegasusForConditionalGeneration
+    from transformers import PegasusModel
+    from transformers import SwinForImageClassification
+    from transformers import SwinForMaskedImageModeling
+    from transformers import SwinModel
+    from transformers import ViTForImageClassification
+    from transformers import ViTForMaskedImageModeling
+    from transformers import ViTModel
     from transformers import XLNetConfig
     from transformers import XLNetLMHeadModel
 
 
-# We are primarily interested in tf32 datatype
-torch.backends.cuda.matmul.allow_tf32 = True
```

Comment:
where did this go? We still want to benchmark with tf32.

Reply:
Ah thanks! I thought I had it in common.py. Added it in common.py now.